### ISSUE TITLE
not working for seekingalpha

### ISSUE THREAD

**Issue by githubfun** [reek/anti-adblock-killer#1633]
`Hey, we noticed you are using an ad blocker`

this is a great script that has worked for me on forbes.com and bloomberg.com. It already has logic to work with seekingalpha.com but somehow I still get the above popup.

**Comment by maynak00** [comment]
no antiadblock for me too (w7+chromium+uBO).cant test on linux
**try with:**
`@@||staticseekingalpha.a.ssl.fastly.net/pages/ads*.js$script,domain=seekingalpha.com`
antiadblock: http://staticseekingalpha.a.ssl.fastly.net/pages/ads_407458.js

**Comment by reek** [comment]
@githubfun 
https://github.com/reek/anti-adblock-killer/wiki/Report-Guide

**Comment by githubfun** [comment]
*http://seekingalpha.com/
*
![2016-06-26-181140_1920x1080_scrot](https://cloud.githubusercontent.com/assets/4968931/16366275/9acf7dee-3bc9-11e6-90a9-85675dabaadc.png)

* google chrome Version 50.0.2661.102 (64-bit) on debian stable linux
* Tampermonkey latest
* ublock origin latest

**Comment by githubfun** [comment]
@matyo91 sorry I am not sure how to test your recipe but I'd like to try it out,  could you advise? Thanks!

**Comment by matyo91** [comment]
Yes, @githubfun i realized that i commited changes since 2 years ago !
It was when playtv.fr changed host to play.tv. I just changed that from original commit.
I looked at the web site, it now use both domains
Some how, it doesn't work any more : 
for instance : http://play.tv/live-tv/9/arte/ or http://playtv.fr/television/arte/

https://github.com/reek/anti-adblock-killer/commit/4a97e86ea2dd37d540be4c5f482871e859c592c7

**Comment by githubfun** [comment]
@matyo91 Thank you! however I realized I @ wrong username, was was meant for @maynak00

**Comment by reek** [comment]
@githubfun 
Not detected for me, but can you test @maynak00  rule
https://github.com/reek/anti-adblock-killer/wiki/Add-filter

**Comment by githubfun** [comment]
@reek @maynak00 Tried adding @maynak00  rule to My filters in Ublock origin, still does not work on linux. 

However, for windows7 + ublock origin + AAK, there was no issue and no additional rule is needed. 

Please advise if I can test further. Thanks!

**Comment by reek** [comment]
@githubfun 
I have no other solution for you
